app/mailers/candidate_mailer.rb

```diff
@@ -22,6 +22,11 @@ def application_submitted(application_form)
 
   def chase_reference(reference)
     @reference = reference
+    @application_form = @reference.application_form
+
+    if @application_form.show_new_reference_flow?
+      @provider = @application_form.application_choices.pending_conditions.first.provider
```

Comment:
This conditional is repeated 4 times in the mailer class and it seems all it does is populate `@provider` for use in the template.
Suggest we just extract it into a variable in the template/partial as we don't use it in these mailer methods do we?